src/driver/odoc.ml

```diff
@@ -1,12 +1,20 @@
 open Bos
 
+type id = Fpath.t
+
+let fpath_of_id id = id
+let id_of_fpath id = id
+
+let index_filename = "index.odoc-index"
+
 type compile_deps = { digest : Digest.t; deps : (string * Digest.t) list }
 
-let odoc = Cmd.v "./_build/default/src/odoc/bin/main.exe"
 (* This is the just-built odoc binary *)
+let default = "./_build/default/src/odoc/bin/main.exe"
```

Comment:
When we run the driver with `dune exec` I think we should be able to call odoc as `odoc`. The reference below could be removed also, as `odoc` should be available in every contextes.

Reply:
Fixed.